docs/src/examples/components/Segment/Variations/SegmentExampleInverted.shorthand.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+import { Segment, Text } from '@stardust-ui/react'
+
+const SegmentExampleInvertedShorthand = () => (
+  <div>
+    <Segment content="Colored segment." variables={siteVars => ({ color: siteVars.brand })} />
+    <br />
+    <Segment
+      inverted
+      content={<Text content="Colored inverted segment." styles={{ color: 'white' }} />}
+      variables={siteVars => ({ color: siteVars.brand })}
```

Comment:
would suggest to use some vivid fallback color here

Reply:
thx, done